Dockerfile

```diff
@@ -31,19 +31,25 @@ ENV EFS_CLIENT_SOURCE=$client_source
 
 RUN GOOS=${TARGETOS} GOARCH=${TARGETARCH} make aws-efs-csi-driver
 
-FROM amazonlinux:2.0.20210219.0
+FROM amazonlinux:2.0.20210421.0
 # Install efs-utils from github by default. It can be overriden to `yum` with --build-arg when building the Docker image.
 # If value of `EFSUTILSSOURCE` build arg is overriden with `yum`, docker will install efs-utils from Amazon Linux 2's yum repo.
 ARG EFSUTILSSOURCE=github
```

Comment:
Should change it back to yum since 1.31.1 is already present in the yum repo of amazon linux?

Did we consider the logic that if the version is not present in yum repo, then we fallback to use the installation of Github?

Reply:
Since it takes a while for new `efs-utils` release to show up on yum, I think we should install it directly from github. 

Also installing from github always ensures latest efs-utils version is installed. We could avoid bumping up the `efs-utils` release version everytime a new version is released.

Reply:
as a middleground, we could checkout latest github tag/version instead of building master branch. something like https://stackoverflow.com/questions/17414104/git-checkout-latest-tag/17414170.

Reply:
+1